tests/legacy-cli/e2e/tests/generate/application/application-zoneless.ts

```diff
@@ -0,0 +1,9 @@
+import { ng } from '../../../utils/process';
+import { useCIChrome } from '../../../utils/project';
+
+export default async function () {
+  await ng('generate', 'application', 'app2', '--experimental-zoneless');
+  await useCIChrome('app2', 'projects/app2');
+  await ng('test', 'app2', '--watch=false');
+  await ng('build', 'app2');
```

Comment:
Maybe it would be better to also add a test for non standalone. 


```suggestion
await ng('generate', 'app', 'standalone', '--experimental-zoneless', '--standalone');
await useCIChrome('standalone', 'projects/standalone');
await ng('test', 'standalone', '--watch=false');
await ng('build', 'standalone');

await ng('generate', 'app', 'ngmodules', '--experimental-zoneless', '--no-standalone', '--skip-install');
await useCIChrome('ngmodules', 'projects/ngmodules');
await ng('test', 'ngmodules', '--watch=false');
await ng('build', 'ngmodules');
```